extensions/binding/org.eclipse.smarthome.binding.iota/src/main/java/org/eclipse/smarthome/binding/iota/handler/IotaBridgeHandler.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.iota.handler;
+
+import java.math.BigDecimal;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.binding.iota.IotaBindingConstants;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.io.iota.utils.IotaUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link IotaBridgeHandler} is responsible for handling the
+ * bridge over IOTA's distributed ledger, the Tangle.
+ *
+ * @author Theo Giovanna - Initial Contribution
+ */
+public class IotaBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(IotaBridgeHandler.class);
+    private int port = 443;
+    private String protocol = "https";
+    private String host = "nodes.testnet.iota.org";
+    private IotaUtils utils;
+
+    public IotaBridgeHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(@NonNull ChannelUID channelUID, @NonNull Command command) {
+        // not needed
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing IOTA bridge");
+        Configuration config = getThing().getConfiguration();
+        setHost(getOrDefault(config.get(IotaBindingConstants.HOST), getHost()));
```

Comment:
Since I removed the default value in the variable initialization, this will just set them to anything different from null if a config value is provided. Otherwise it'll stay as it is.

Reply:
Fixed